frontend/ui/elements/filemanager_menu_order.lua

```diff
@@ -67,6 +67,7 @@ local order = {
     },
     screen = {
         "screensaver",
+        "screenshot",
```

Comment:
I think the Device menu might make more sense than the Screen menu, but I'm not really sure.

Reply:
No strong opinion on that.
But we might visit the `Screen>` menu more often than the `Device>` one, so, as something of little regular use, I'd put it in `Device>` with the other misc things.

Reply:
I also thought about put it in `Device>`.